cdap-ui/app/directives/log-viewer/log-viewer.html

```diff
@@ -127,12 +127,11 @@
         </th>
       </thead>
       <tbody infinite-scroll="LogViewer.scrollFn()"
-             infinite-scroll-disabled="LogViewer.viewLimit >= LogViewer.displayData.length"
```

Comment:
I think we might still need this condition. What happens when the program has stopped running and we have reached the end of the logs. We want to disable the infinite scroll because there is nothing to scroll I believe.